doc/Handlers.md

```diff
@@ -0,0 +1,415 @@
+      "payload_hash": "2f338fa547652b58b267669c37dd477d4ea9eecf4b4027c2ea996421ce378082",
+      "state_root_hash": "7158d5b36d2f7ac85c84d2e5c46e50479757c9e3769d67e3b7a72a114e316776",
+      "withdrawal_handles_hash": "0e5751c026e543b2e8ab2eb06099daa1d1e5df47778f7787faab45cdf12fe3a8",
+      "validators_hash": "d139b7cda8577e16c79084a253d6e9ad5cfeb07e2e5ad46223fa0f87c500b9d8",
+      "previous_hash": "0486b61be7f424f6f2ede3f5dbd0c68990e7dce51e26463b73bade7767f6be68",
+      "author": "tz1PmKMKTbSVcam69ssKUtmSpeBRkYPTBPvW",
+      "block_height": 2148,
+      "operations": []
+    },
+    {
+      "hash": "0486b61be7f424f6f2ede3f5dbd0c68990e7dce51e26463b73bade7767f6be68",
+      "payload_hash": "751be9f3529084c4fc34f06b87c1a36477357b6412052abb2542952af35955ff",
+      "state_root_hash": "7158d5b36d2f7ac85c84d2e5c46e50479757c9e3769d67e3b7a72a114e316776",
+      "withdrawal_handles_hash": "0e5751c026e543b2e8ab2eb06099daa1d1e5df47778f7787faab45cdf12fe3a8",
+      "validators_hash": "d139b7cda8577e16c79084a253d6e9ad5cfeb07e2e5ad46223fa0f87c500b9d8",
+      "previous_hash": "5d6109c326976a0b831b0a613b7174599c19d9f215d532bcbcacb161f8dec644",
+      "author": "tz1PmKMKTbSVcam69ssKUtmSpeBRkYPTBPvW",
+      "block_height": 2147,
+      "operations": []
+    }
+  ],
+  "last_block": {
+    "hash": "cf73d4db3c4640116651d23d6791c60a872c9d74f9ee803fc3067a89f2f9cd85",
+    "payload_hash": "040f080a0833bdf14b2b04a857d39ae91c0597d5f759b68add1a353c3fc9aa99",
+    "state_root_hash": "7158d5b36d2f7ac85c84d2e5c46e50479757c9e3769d67e3b7a72a114e316776",
+    "withdrawal_handles_hash": "0e5751c026e543b2e8ab2eb06099daa1d1e5df47778f7787faab45cdf12fe3a8",
+    "validators_hash": "d139b7cda8577e16c79084a253d6e9ad5cfeb07e2e5ad46223fa0f87c500b9d8",
+    "previous_hash": "28f8c0e42b627e9357f65c3d759e26e36cbf5e7d1c95fb1326f38477cd46b4c6",
+    "author": "tz1PmKMKTbSVcam69ssKUtmSpeBRkYPTBPvW",
+    "block_height": 2149,
+    "operations": []
+  },
+  "last_block_signatures": [
+    {
+      "signature": "edsigtamTx6dHvcRg9oT3kMr1R8UpNGwuKVJZ4mMKHddkiJwRH1k3ZjBwHE6U9b6JcG765TQvnxnYGZKj2R1nYJCtmvLDWYE4re",
+      "public_key": "edpkuqseqRnfW5ZWwpp2kWjdY2Tma8A8xhNND2Xgoos49gApQb46LP"
+    },
+    {
+      "signature": "edsigtqw2eeXbeDXrgumtP7tG5SroLnvHsPmeLmJtPvuo6bbqSP5VsBdvvKkRH2U146nwMTvTgj9dBJhGem9QH2QJJbzh2Vosxc",
+      "public_key": "edpkuy1Qjwo4X7a3r1wA1HLAx9s5kgp7b9xUxfSpQQcTpFnqyzyrzH"
+    }
+  ]
+}
+```
+
+## `/request-nonce`
+
+### Purpose
+
+??
+
+### Expected data
+
+A URI?
+
+### Returned data
+
+The nonce of type `BLAKE2B.t`
+
+### curl example
+
+```shell script
+$
+```
+
+## `/register-uri`
+
+### Purpose
+
+??Add the provided URI as a new validator??
+
+### Expected data
+
+- A URI
+- The corresponding signature
+
+### curl example
+
+```shell script
+$
+```
+
+## `/user-operation-gossip`
+
+### Purpose
+
+Add a user operation to the pending operations.
+If the operation does not already exist, it is broadcasted to the gossip network
```

Comment:
`If the operation does not exist, it is then broadcast to the gossip network`

Reply:
Done.